rundeck-data-model/src/main/java/org/rundeck/app/data/model/v1/execution/RdReferencedExecution.java

```diff
@@ -0,0 +1,10 @@
+package org.rundeck.app.data.model.v1.execution;
+
+
+public interface RdReferencedExecution {
+
+    String getStatus();
+    Long getExecutionId();
+    Long getScheduledExecutionJobId();
```

Comment:
This should be renamed to `getJobId()`. Let's use `Serializable` as the type instead of forcing a Long as the implementation.